mmv1/third_party/terraform/services/datalossprevention/resource_data_loss_prevention_discovery_config_test.go

```diff
@@ -1292,3 +1365,249 @@ resource "google_data_loss_prevention_discovery_config" "basic" {
 }
 `, context)
 }
+
+func testAccDataLossPreventionDiscoveryConfig_dlpDiscoveryConfigStartAwsS3(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+data "google_project" "project" {
+}
+resource "google_organization_iam_member" "service_agent" {
+	org_id = "%{organization}"
+	role    = "roles/servicemanagement.admin"
+	member = "serviceAccount:service-${data.google_project.project.number}@dlp-api.iam.gserviceaccount.com"
+}
+resource "google_organization_iam_member" "org_admin" {
+	org_id = "%{organization}"
+	role    = "roles/resourcemanager.organizationAdmin"
+	member = "serviceAccount:service-${data.google_project.project.number}@dlp-api.iam.gserviceaccount.com"
+}
+resource "google_organization_iam_member" "dlp_role" {
+    org_id = "%{organization}"
+	role    = "roles/dlp.orgdriver"
```

Comment:
nit: there's some weird indentation in the test configs (GitHub doesn't show it, but I'll bet it's due to a combination of tabs and spaces)